presto-main/src/main/java/io/prestosql/execution/SqlQueryManager.java

```diff
@@ -313,4 +323,21 @@ private void enforceCpuLimits()
             }
         }
     }
+
+
+    /**
+     * Enforce query scan bytes limits
+     */
+    private void enforceScanLimits()
+    {
+        for (QueryExecution query : queryTracker.getAllQueries()) {
+            DataSize limit = getQueryMaxScanPhysicalBytes(query.getSession());
```

Comment:
There should be a config limit in QueryManagerConfig as well.
But we shouldn't allow users to bypass the server config limit through session override.
Can follow this PR for those changes https://github.com/prestosql/presto/pull/2083/files

Reply:
done